aws-shared-1/main.tf

```diff
@@ -2,22 +2,23 @@ variable "bastion_ami" { default = "ami-53d4a344" }
 variable "env" { default = "shared" }
 variable "index" { default = "1" }
 variable "nat_ami" { default = "ami-12c5b205" }
-variable "public_subnet_1b" { default = "10.10.1.0/24" }
-variable "public_subnet_1e" { default = "10.10.4.0/24" }
+variable "public_subnet_1b_cidr" { default = "10.10.1.0/24" }
```

Comment:
for clarification -- we're putting "cidr" instead of just "subnet" to be 100% explicit that we're not talking about the subnet ID, right?